datatypes/mcs_datatype.h

```diff
@@ -248,6 +249,7 @@ class SystemCatalog
         @brief Convenience method to get int128 from a std::string.
     */
     int128_t decimal128FromString(const std::string& value, bool* saturate = 0) const;
+    int128_t decimal128FromString(const utils::NullString& value, bool* saturate = 0) const;
```

Comment:
Do you think it is viable to remove the first overload where std::string is used?

Reply:
Let me try.

Reply:
It is unviable. Too many changes required.